test/TestFunctional.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+import subprocess
+import os
+# import re
+from itertools import combinations, product, chain
+
+
+class TestFunctional:
+    __test__ = False
+    """Tests for the overall functionality of the Wochenende pipeline.
+
+    Attributes
+    ----------
+    _p_fixed_arguments : dict [str, str]
+        Fixed arguments that need to be set like this for testing. Value is empty if the
+        argument just needs to exist
+    _p_options : dict [str, list of str]
+        Optional arguments that take an input. Each list gives the inputs for testing.
+    _p_flags : list of str
+        Optional arguments that do not require an additional input.
+
+    Notes
+    -----
+    To run only a few combinations, just comment the irrelevant ones.
+    This test class relies on an existing slurm installation. It might be too much to run
+    this due to combinatorial explosion. For a simple test of installation, one can rely
+    on `test_installation.py`. 
+    """
+
+    # TODO: PE support
+    # TODO: Test partially successful runs, continuation and --force-restart
+    # TODO: Collect test run information
+    # TODO: Sum up test run information
+
+    _p_fixed_arguments = [
+        '--metagenome', 'test',
+        '--debug'
+    ]
+    _p_options = {
+        '--aligner': ['bwamem', 'minimap2', 'ngmlr'],
```

Comment:
there are now minimap2short and minimap2long as aligner. You'll need to add both here.